mapstruct/src/main/java/com/baeldung/entity/Document.java

```diff
@@ -0,0 +1,15 @@
+package com.baeldung.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Date;
+
+@Getter
+@Setter
```

Comment:
While I know what these lombok annotations are doing, it's probably easier not to mix Lombok into this article on another subject - explicit getters/setters is more transparent. Your IDE can generate them, and then the reader can see them.